### PR TITLE
Auth add providers 23.06.2023

### PR DESCRIPTION
**Pull request by arifm6**
Set up backend providers for github and google. One potential issue is that duplicate emails on different providers is not allowed. This is standard in a lot of authentication systems.

### CODE REVIEW

File: src/pages/home.tsx
Comment (nicitaacom) [inline src/pages/home.tsx:1]: Why you created home.tsx - we don't need it

File: src/pages/index.tsx
Comment (nicitaacom) [inline src/pages/index.tsx:1]: Specify const name please
const hello - doesn't tell me anything

Comment (nicitaacom) [reply src/pages/index.tsx:1]: Also infinity tRPC loading
image

**Review by nicitaacom — CHANGES_REQUESTED** [review]
Fontend design required
For now its fine but facebook provider needed (so do auth with facebook)
image

Also I would like name context as context - not with ctx

Delete my auth with github provider in database - so I can test how auth with google works
image

File: src/pages/index.tsx
Comment (arifm6) [reply src/pages/index.tsx:1]: this is just for testing. hello is the greeting msg.

**Review by nicitaacom — APPROVED** [review]
I approve it - so we will up to date with development branch
Auth with google and github works fine - fix with facebook needed

**Pull request merged**